projects/engine/doc/connector/plugin-connector.md

```diff
@@ -0,0 +1,92 @@
+# Publish a Plugin Connector
```

Comment:
should that be `Develop a Plugin Connector` instead ? where would the connector published?

Reply:
On npm. I'll add `Develop & Publish`.